### PR TITLE
i18n(de): update translation guides/authoring-content

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

Translation of `guides/authoring-content.mdx` into German language.

### CODE REVIEW

**Comment by trueberryless** [comment]
Link fixed in https://github.com/withastro/starlight/pull/2458

**Comment by trueberryless** [comment]
Link fixed in https://github.com/withastro/starlight/pull/2459

File: docs/src/content/docs/de/guides/authoring-content.mdx [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+</details>
+```
+
+## Andere allgemeine Markdown-Funktionen
+
+Starlight unterstützt alle anderen Markdown-Autorensyntaxen, wie Listen und Tabellen. Einen schnellen Überblick über alle Markdown-Syntaxelemente findest du im [Markdown Cheat Sheet von The Markdown Guide](https://www.markdownguide.org/cheat-sheet/).
+
+## Erweiterte Markdown- und MDX-Konfiguration
+
+Starlight verwendet Astros Markdown- und MDX-Renderer, der auf remark und rehype aufbaut. Du kannst eine Unterstützung für eigene Syntax und Verhalten hinzufügen, indem du `remarkPlugins` oder `rehypePlugins` in deiner Astro-Konfigurationsdatei hinzufügst. Weitere Informationen findest du unter ["Markdown konfigurieren"] (https://docs.astro.build/de/guides/markdown-content/#markdown-konfigurieren) in der Astro-Dokumentation.
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/authoring-content.mdx:395]:
```suggestion
Starlight verwendet Astros Markdown- und MDX-Renderer, der auf remark und rehype aufbaut. Du kannst eine Unterstützung für eigene Syntax und Verhalten hinzufügen, indem du `remarkPlugins` oder `rehypePlugins` in deiner Astro-Konfigurationsdatei hinzufügst. Weitere Informationen findest du unter ["Markdown konfigurieren"](https://docs.astro.build/de/guides/markdown-content/#markdown-plugins) in der Astro-Dokumentation.
```

Extra space in the link and update the anchor.

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for another translation.

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://contribute.docs.astro.build/reviewers/reviewing-translations/) and except for 1 link formatting issue, this looks great to me :thumbsup:

**Comment by trueberryless** [comment]
Errare humanum est! Thanks for the review, should be good to go!

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the PR, everything looks good to me now :rocket: I'll temporarily merge with broken links so they can be fixed in the next PR.

**Pull request merged**